src/loaders/OMETiffLoader.js

```diff
@@ -156,110 +29,34 @@ export default class OMETiffLoader {
-   * @param {Array} args.loaderSelection, Array of number Arrays specifying channel selections
-   * @returns {Object} data: TypedArray[], width: number, height: number
-   * Default is `{data: [], width, height}`.
-   */
-  async getRaster({ z, loaderSelection }) {
-    const { tiff, omexml, isBioFormats6Pyramid, pool, isInterleaved } = this;
-    const { SizeZ, SizeT, SizeC } = omexml;
-    const rasters = await Promise.all(
-      loaderSelection.map(async sel => {
-        const index = this._getIFDIndex(sel);
-        const pyramidIndex = z * SizeZ * SizeT * SizeC + index;
-        // We need to put the request for parsing the file directory into this array.
-        // This allows us to get tiff pages directly based on offset without parsing everything.
-        if (!isBioFormats6Pyramid) {
-          this._parseIFD(pyramidIndex);
-        } else {
-          // Pyramids with large z-stacks + large numbers of channels could get slow
-          // so we allow for offsets for the initial images ("parentImage").
-          this._parseIFD(index);
-          const parentImage = await tiff.getImage(index);
-          if (z !== 0) {
-            tiff.ifdRequests[pyramidIndex] = tiff.parseFileDirectoryAt(
-              parentImage.fileDirectory.SubIFDs[z - 1]
-            );
-          }
-        }
-        const image = await tiff.getImage(pyramidIndex);
-        // Flips bits for us for endianness.
-        const raster = await image.readRasters({
-          pool,
-          interleave: isInterleaved
-        });
-        return isInterleaved ? raster : raster[0];
-      })
-    );
-    // Get first selection * SizeZ * SizeT * SizeC + loaderSelection[0] size as proxy for image size.
-    if (!loaderSelection || loaderSelection.length === 0) {
-      return { data: [], ...this.getRasterSize({ z }) };
-    }
-    const image = await tiff.getImage(
-      z * SizeZ * SizeT * SizeC + this._getIFDIndex(loaderSelection[0])
-    );
-    const width = image.getWidth();
-    const height = image.getHeight();
+  async getTile(args) {
+    const tiles = await super.getTile(args);
+    return this._convertIntRasters(tiles);
+  }
 
-    let data;
-    if (this.dtype === '<f4') {
-      // GeoTiff.js returns 32 bit uint when the tiff has 32 significant bits.
-      data = rasters.map(r => new Float32Array(r.buffer));
-    } else if (this.omexml.Type.startsWith('int')) {
-      // geotiff.js returns the correct typedarray but need to cast to Uint for viv.
-      const b = rasters[0].BYTES_PER_ELEMENT;
-      // eslint-disable-next-line no-nested-ternary
-      const T = b === 1 ? Uint8Array : b === 2 ? Uint16Array : Uint32Array;
-      data = rasters.map(r => new T(r));
-    } else {
-      data = rasters;
-    }
-    return {
-      data,
-      width,
-      height
-    };
+  async getRaster({ z, loaderSelection }) {
+    const rasters = await super.getRaster({
+      z,
+      loaderSelection
+    });
+    return this._convertIntRasters(rasters);
```

Comment:
`super` is common for calling `static` methods on an inherited class not instance methods. I'd use `this`.

```suggestion
    const rasters = await this.getRaster({
      z,
      loaderSelection
    });
    return this._convertIntRasters(rasters);
```